OsmAnd/src/net/osmand/plus/resources/ResourceManager.java

```diff
@@ -26,8 +26,8 @@
 import net.osmand.PlatformUtil;
 import net.osmand.ResultMatcher;
 import net.osmand.binary.BinaryMapIndexReader;
-import net.osmand.binary.BinaryMapIndexReader.SearchPoiTypeFilter;
 import net.osmand.binary.BinaryMapIndexReader.SearchPoiAdditionalFilter;
+import net.osmand.binary.BinaryMapIndexReader.SearchPoiTypeFilter;
```

Comment:
I've noticed that there a lot superficial changes  (probably done by your IDE) which only change the orders or imports. This is just one example, see https://github.com/osmandapp/OsmAnd/pull/20016/files?diff=split&w=1 for full list.

Those enlarge the diff needlessly, and make it harder to review changes. Would it be possible that you clean those up?

Reply:
Hi, the merge conflicts are resolved, imports reorganized and the apk is here: https://github.com/KnollFrank/OsmAnd/releases/download/testing/OsmAnd-nightlyFree-legacy-fat-debug.apk